server/boot/home.js

```diff
@@ -37,7 +37,7 @@ module.exports = function(app) {
     const referer = req.headers.referer;
```

Comment:
that's neat! I'll initialize it with `''` instead of `'/'` because the value is not exist